src/modifiers/detectOverflow.js

```
@@ -31,9 +31,15 @@ export function detectOverflow(
   const documentElement = options.boundaryElement.ownerDocument.documentElement;
 
   if (!options.boundaryElement.contains(popperElement)) {
-    console.error(
```

I think the library should throw an error also in production, this is a pretty important error that shouldn't be ignored.

I know React has shorted error messages for production builds, could we do the same?

---

Maybe a dumb idea but we could have an error code for PROD, and it links to a file that explains each code. That way there's no words to bloat the bundle. 

e.g.

```
Popper: Error 5 {link}
```

I'm not sure how practical that would be to maintain

---

React uses bit.ly-like links to point to a documentation page, that would work too